### PR TITLE
Added ALERT pin functionality

### PR DESCRIPTION
**Pull request by m0ivq**
Added ALERT pin specific functions.

### CODE REVIEW

**Comment by m0ivq** [comment]
All travis errors now fixed.

**Review by siddacious — CHANGES_REQUESTED** [review]
Awesome @m0ivq ! Thanks for your work, this looks great. I'd like to test it before merging; Could I trouble you to write a simple sketch that exercises the new code? Ideally with some setup instructions. It needn't be a magnum opus, just something like whatever you used to test as you were writing the code.

Thank you! This would help me test the code quicker and get it merged ASAP.

**Comment by dougie181** [comment]
I have tested the alert functionality by setting the OVERVOLTAGE alert and it worked fine. Thanks @m0ivq for your great work!

In my scenario, I use the alert to trigger an external interrupt to wakeup when the voltage is over 3v.

```
boolean initINA260Sensor() {

  if (!ina260.begin()) {
     Serial.println("Couldn't find INA260 chip");
     return false;
  }

  // setup the alert functionality
  ina260.setAlertLimit(3000);
  ina260.setAlertType(INA260_ALERT_OVERVOLTAGE);

  return true;
}
```

File: library.properties [hunk 1]
```diff
@@ -1,10 +1,9 @@
 name=Adafruit INA260 Library
-version=1.3.5
+version=1.3.2
```
Comment (siddacious) [inline library.properties]: Looks like an old version; This should be bumped to 1.4.0 and the dependencies should remain

**Review by siddacious — CHANGES_REQUESTED** [review]
I only left one specific comment but it looks like you need to merge `adafruit/master` into your branch.

The `.github` directory should stay and the travis config should not be resurrected ;)

At first glance it code looks good so please fix the above and I will test. 

Thank you for your contribution! This was one is close to my <3

**Comment by siddacious** [comment]
@m0ivq Please also add and example that demonstrates the new functionality. That will help with testing and will help users to use your contribution.

Thank you!

**Comment by m0ivq** [comment]
@siddacious something went wrong when I merged into my branch so reverted it back but ran out of time to try again. I'll try again on Monday once I've added a few examples. I can confirm that overcurrent is definitely working correctly as it's been in use daily in a test fixture at my work since the original PR!

**Comment by m0ivq** [comment]
I've cleaned up the PR but don't have access to hardware to test at the moment. @siddacious it appears that some bits got out of sync between commits on the branches but I've manually put back the changes that were refusing to merge from adafruit/master!

**Comment by ladyada** [comment]
please run clang-format again - reply when the PR is passing CI

**Pull request closed without merging**